handler.go

```diff
@@ -28,17 +24,16 @@ func (v *verifier) handleCheckImages(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if len(requestData.Images) == 0 {
-		log.Printf("missing images in %v", requestData)
-		http.Error(w, "missing required parameter 'images'", http.StatusNotAcceptable)
-		return
-	}
-
-	ctx := context.Background()
-	data, err := v.verifyImages(ctx, requestData.Images)
-	if err != nil {
-		http.Error(w, err.Error(), http.StatusInternalServerError)
-		return
+	data := make([]byte, 0)
+	if reflect.ValueOf(requestData.Images).IsZero() {
+		v.logger.Infof("images variable not found")
```

Comment:
Should we consider this as an error?

https://github.com/nirmata/kyverno-notation-aws/pull/6/files#diff-0eb779b9e49d8e44b0f36923fdb8d87d5ee024f886eefc45deec4ec88380a087L31-L35